packages/orbit-components/src/common/tailwind/spacing.ts

```diff
@@ -177,29 +177,276 @@ export const verticalTokens = {
   },
 };
 
+export const gapTokens = {
+  [SPACING.none]: "gap-none",
+  [SPACING.XXXSmall]: "gap-xxxs",
+  [SPACING.XXSmall]: "gap-xxs",
+  [SPACING.XSmall]: "gap-xs",
+  [SPACING.small]: "gap-sm",
+  [SPACING.medium]: "gap-md",
+  [SPACING.large]: "gap-lg",
+  [SPACING.XLarge]: "gap-xl",
+  [SPACING.XXLarge]: "gap-xxl",
+  [SPACING.XXXLarge]: "gap-xxxl",
+  [QUERIES.MEDIUMMOBILE]: {
+    [SPACING.none]: "mm:gap-none",
+    [SPACING.XXXSmall]: "mm:gap-xxxs",
+    [SPACING.XXSmall]: "mm:gap-xxs",
+    [SPACING.XSmall]: "mm:gap-xs",
+    [SPACING.small]: "mm:gap-sm",
+    [SPACING.medium]: "mm:gap-md",
+    [SPACING.large]: "mm:gap-lg",
+    [SPACING.XLarge]: "mm:gap-xl",
+    [SPACING.XXLarge]: "mm:gap-xxl",
+    [SPACING.XXXLarge]: "mm:gap-xxxl",
+  },
+  largeMobile: {
+    [SPACING.none]: "lm:gap-none",
+    [SPACING.XXXSmall]: "lm:gap-xxxs",
+    [SPACING.XXSmall]: "lm:gap-xxs",
+    [SPACING.XSmall]: "lm:gap-xs",
+    [SPACING.small]: "lm:gap-sm",
+    [SPACING.medium]: "lm:gap-md",
+    [SPACING.large]: "lm:gap-lg",
+    [SPACING.XLarge]: "lm:gap-xl",
+    [SPACING.XXLarge]: "lm:gap-xxl",
+    [SPACING.XXXLarge]: "lm:gap-xxxl",
+  },
+  tablet: {
+    [SPACING.none]: "tb:gap-none",
+    [SPACING.XXXSmall]: "tb:gap-xxxs",
+    [SPACING.XXSmall]: "tb:gap-xxs",
+    [SPACING.XSmall]: "tb:gap-xs",
+    [SPACING.small]: "tb:gap-sm",
+    [SPACING.medium]: "tb:gap-md",
+    [SPACING.large]: "tb:gap-lg",
+    [SPACING.XLarge]: "tb:gap-xl",
+    [SPACING.XXLarge]: "tb:gap-xxl",
+    [SPACING.XXXLarge]: "tb:gap-xxxl",
+  },
+  [QUERIES.DESKTOP]: {
+    [SPACING.none]: "de:gap-none",
+    [SPACING.XXXSmall]: "de:gap-xxxs",
+    [SPACING.XXSmall]: "de:gap-xxs",
+    [SPACING.XSmall]: "de:gap-xs",
+    [SPACING.small]: "de:gap-sm",
+    [SPACING.medium]: "de:gap-md",
+    [SPACING.large]: "de:gap-lg",
+    [SPACING.XLarge]: "de:gap-xl",
+    [SPACING.XXLarge]: "de:gap-xxl",
+    [SPACING.XXXLarge]: "de:gap-xxxl",
+  },
+  [QUERIES.LARGEDESKTOP]: {
+    [SPACING.none]: "ld:gap-none",
+    [SPACING.XXXSmall]: "ld:gap-xxxs",
+    [SPACING.XXSmall]: "ld:gap-xxs",
+    [SPACING.XSmall]: "ld:gap-xs",
+    [SPACING.small]: "ld:gap-sm",
+    [SPACING.medium]: "ld:gap-md",
+    [SPACING.large]: "ld:gap-lg",
+    [SPACING.XLarge]: "ld:gap-xl",
+    [SPACING.XXLarge]: "ld:gap-xxl",
+    [SPACING.XXXLarge]: "ld:gap-xxxl",
+  },
+};
+
+const safariVerticalTokens = {
+  [SPACING.none]: "safari:[&>*:not(:last-child)]:safari-y-none",
```

Comment:
I'm not sure I understand this. I understand the `safari:` that will apply `@supports (-webkit-touch-callout: none) and (not (translate: none))` but then it should be `space-y-none`, right? I can't test it effectively, because I don't have the device and browser, but I think this is a typo, right?

Reply:
right, it's typo, thanks 👍🏻